app/assets/stylesheets/components/_carousel_arrows.scss

```diff
@@ -0,0 +1,11 @@
+
+@keyframes flash {
+  from {background-color: transparent;}
+  to {background-color: orange;}
+}
+
+.arrow-cust:hover {
+  animation-name: flash;
+  animation-duration: 1s;
+  animation-iteration-count: 15;
+}
```

Comment:
when you hover over the arrows in carousel, runs a cheesy animation like car indicators.

Reply:
Cant' wait to see it !